### PR TITLE
angular-mapexplorer: import angular-mapexplorer

### PR DESCRIPTION
**Pull request by such**

### CODE REVIEW

File: packages/angular-mapexplorer/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+before_install:
+  - export CHROME_BIN=chromium-browser
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - npm update -g
+before_script:
+  - npm install -qg bower
+  - bower install
+script:
+  - npm run test:ci
```
Comment (t-bast) [inline packages/angular-mapexplorer/.travis.yml]: Will it work (having a travis.yml file at the package level)? Or should we remove this one and let the root level one handle this?

File: packages/angular-mapexplorer/package.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    "build:npm:umd": "rollup -c rollup.umd.config.js",
+    "build:browser": "rollup -c rollup.browser.config.js",
+    "build:demo": "rollup -c rollup.browser.demo.config.js",
+    "build": "npm run build:npm && npm run build:browser",
+    "prepublish": "npm run build",
+    "update-license-headers": "./update-license-headers.sh"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stratumn/angular-mapexplorer"
```
Comment (t-bast) [inline packages/angular-mapexplorer/package.json]: nit: should be updated to stratumn/indigo-js

File: packages/angular-mapexplorer/update-license-headers.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+for d in $directories; do
+	if [ -d "$d" ]; then
+		for e in $extensions; do
+			for f in $(find $d -name "*.$e"); do
+				update-license-header $f
+			done
+		done
+	fi
+done
```
Comment (t-bast) [inline packages/angular-mapexplorer/update-license-headers.sh]: nit: probably not needed anymore (we have the root level one)

**Comment by such** [comment]
@t-bast thanks good points!

**Pull request merged**